static/src/javascripts/lib/url.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import detect from 'lib/detect';
+
+const supportsPushState = detect.hasPushStateSupport();
+
+// returns "foo=bar&fizz=buzz" (eg. no ? symbol)
+function getCurrentQueryString(): string {
+    return window.location.search.replace(/^\?/, '');
+}
+
+// returns a map of querystrings
+// eg ?foo=bar&fizz=buzz returns {foo: 'bar', fizz: 'buzz'}
+function getUrlVars(options?: Object = {}): Object {
```

Comment:
This function receives an object with one property (the `query` string). It is called with the `options` argument, if I am not mistaken, in [one place in the codebase](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/discussion/upvote.js#L80)

Can we refactor this function to optionally accept a `query` string, to make it clearer? It would be nice if we could destructure the argument, but I don't think the syntax exists yet.